framework/src/modules/random/utils.ts

```diff
@@ -77,14 +73,14 @@ export const getRandomSeed = (
 		throw new Error('Number of seeds cannot be greater than 1000.');
```

Comment:
I guess this checked would be removed in a separate PR?

Reply:
Correct, it is handled in #8575. I haven't selected you as the reviewer of that one, because that change has nothing to do with the protocol.

The main reason to handle it in a separate PR is that this PR is a protocol change, and as such should be merged to `release` branch. The removal of 1000 check is an improvement, which we merge to `development` branch.